File: 151003/Yagnnish/Lab1/Dtos/ResponseDtos/StoryResponseDto.cs
namespace lab_1.Dtos.ResponseDtos
{
    public class StoryResponseDto:BaseResponseDto
    {
        public long? authorId { get; set; }
        public string? title { get; set; }
        public string? content { get; set; }
        public DateTime created { get; set; }
        public DateTime modified { get; set; }
    }
}

File: 151003/Haritonenko/task1/src/main/java/by/haritonenko/rest/dto/response/StoryResponseDto.java
package by.haritonenko.rest.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class StoryResponseDto {

    private Long id;
    private Long authorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}
